manim/mobject/three_dimensions.py

```diff
@@ -3,7 +3,7 @@
 __all__ = ["ThreeDVMobject", "ParametricSurface", "Sphere", "Cube", "Prism"]
 
 from ..constants import *
-from ..mobject.geometry import Square
+from ..mobject.geometry import Square, Dot, Line
 from ..mobject.types.vectorized_mobject import VGroup
 from ..mobject.types.vectorized_mobject import VMobject
 from ..utils.iterables import tuplify
```

Comment:
Can you change here
from ..utils.color import BLUE_D, BLUE, BLUE_E, LIGHT_GREY
to 
from ..utils.color import *
?
Otherwise, the tests are failing and the scene is not running.

Reply:
Done